tests/template_tests/utils.py

```diff
@@ -57,6 +60,8 @@ def inner(self):
                 loaders=loaders,
             )
             func(self)
+            if test_once:
```

Comment:
Nice solution to https://github.com/django/django/pull/3660#issuecomment-65807275